src/application/tester.go

```diff
@@ -55,7 +56,9 @@ func (a *Tester) RunAppTests() error {
 	if numFailed == 0 {
 		return a.Logger.Log("exo-test", "All tests passed", true)
 	}
-	return a.Logger.Log("exo-test", fmt.Sprintf("%d tests failed", numFailed), true)
+	err := a.Logger.Log("exo-test", fmt.Sprintf("%d tests failed", numFailed), true)
+	os.Exit(1)
```

Comment:
Please no exiting from helpers. We should only have exit statements in the `cmd` directory. Please update these methods to return a bool of whether or not the tests ran successfully.